replica/table.cc

```diff
@@ -753,6 +780,7 @@ table::stop() {
                             _main_sstables = _compaction_strategy.make_sstable_set(_schema);
                             _maintenance_sstables = make_maintenance_sstable_set();
                             _sstables = make_compound_sstable_set();
+                            _sstables_changed.signal();
```

Comment:
Why this? It happens way after the flush is complete and compaction_manager is disassociated.

Reply:
let's say we've flush waiting on L0 sstables to drop. then user asks for shutdown. we need to signal the condition variable once the sstable set is cleared in table::stop(), no?

Reply:
although now we have the predicate in condition_variable::wait() and that should result in reactor waking it up